tests/e2e-install-deps.sh

```diff
@@ -264,6 +264,22 @@ remove_golang_os_packages() {
     apt-get --purge autoremove -y
 }
 
+install_libzstd_os_packages() {
+    case $ID in
+    "ubuntu")
+        wait_for_apt_locks
+        apt-get install -y libzstd-dev
+    ;;
+    "almalinux")
+        yum install -y libzstd-devel
+    ;;
+    *)
+        echo "Unsupported OS: $ID"
+        exit 1
+    ;;
+    esac
+}
+
```

Comment:
All this would be unnecessary if we opt for an easier way of updating the AMIs in a bunch. Anyway, after its future upgrade, those steps should be commented/removed.